src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -52,8 +52,7 @@ public static void tearDownAfterClass() throws IOException {
     public void testArgsParsingMainnet() {
         String[] args = {
                 "-p", "14000",
-                "-u", "13000",
-                "-t", "27000",
+                "-nsp", "13000",
```

Comment:
To reflect changes made in b75deb4
```suggestion
                "--neighboring-socket-port", "13000",
```